backend/src/services/group/group.entity.ts

```diff
@@ -1,44 +1,59 @@
 import { getRandomId } from '~/helpers/helpers';
+import { EAMGroupRelatedItem } from '~/common/types/types';
 
 class Group {
   public id: string;
   public name: string;
-  public createdAt: Date;
+  public createdAt: string;
   public tenantId: string;
+  public users?: EAMGroupRelatedItem[];
+  public permissions?: EAMGroupRelatedItem[];
```

Comment:
Ideally entity should contain only other entities or ids. Using dtos in entity is bad because dtos change very often. Entities are more stable. It means that you will have to edit entity very often just because dto changed